apps/homepage/src/app/[lang]/_shared/menuButton/menuButton.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+import styles from "./menuButton.module.scss";
+import Link from "next/link";
+import { motion } from "framer-motion";
+
+type menuButtonProps = {
+  text: string;
+  href: string;
+};
+
```

Comment:
It would be better if we used the children property, I think it's a better option

Should be something like this :

`type menuButtonProps = {
  children: React.ReactNode;
  href: string;
};

Then you pass the children inside your component { children }